code/modules/projectiles/guns/shotguns.dm

```diff
@@ -247,6 +247,65 @@ can cause issues with ammo types getting mixed up during the burst.
 	recoil = RECOIL_AMOUNT_TIER_3
 	recoil_unwielded = RECOIL_AMOUNT_TIER_1
 
+//-------------------------------------------------------------
+//SHOCKGUN - Non-Lethal Fast firing shotgun
+/obj/item/weapon/gun/shotgun/es7
+	name = "\improper ES-7 Supernova Electrostatic Shockgun"
+	desc = "The Weyland-Yutani ES-7 Supernova Electrostatic Shockgun, a Dual-Action shotgun with a quick fire rate."
```

Comment:
```suggestion
	desc = "An archaic electrostatic 20ga shotgun design based on old Earth designs, albeit modernized for its time period. Despite receiving very little upgrades over its service period both within the Weyland Corporation and later the Weyland-Yutani Corporation, it remains popular with mercenaries and security firms because it was designed with security and law enforcement use in mind. Being a dual-mode system, it is capable of firing semi-auto and pump-action modes, although this particular model is strictly semi-auto only. It can only accept X21 slugs."
```'
I said "archaic" specifically because the crew of Prometheus (set in the 2080s) used it.